### PR TITLE
Update Ukrainian Translation

### PR DESCRIPTION
**Pull request by ZaHaDumGoth**
Rest of empty strings and some corrections of fuzzy.

### CODE REVIEW

File: files/lang/uk.po [hunk 1]
```diff
@@ -3886,81 +3879,83 @@ msgstr ""
...
 msgid ""
 "If this checkbox is checked, the event will trigger only once. If not "
 "checked, the event will trigger every time one of the specified players "
 "crosses the event tile."
 msgstr ""
+"Якщо цей прапорець встановлено, подія спрацює лише один раз. Якщо прапорець "
+"не встановлено, подія спрацьовуватиме щоразу, коли один із зазначених "
+"гравців перетинатиме мітку події."
 
 #, fuzzy
```
Comment (ihhub) [inline files/lang/uk.po:3953]: Should we remove this line?

File: files/lang/uk.po [hunk 2]
```diff
@@ -4076,7 +4070,7 @@ msgid "Are you sure you want to overwrite the existing map?"
 msgstr "Ви впевнені, що хочете перезаписати збереження з такою назвою?"
 
 msgid "Map saved to: "
-msgstr ""
+msgstr "Мапу збережено в:"
```
Comment (ihhub) [inline files/lang/uk.po:4073]: Please add a space at the end of line.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @ZaHaDumGoth , I left 2 comments here. Can you please check them?

File: files/lang/uk.po (hunk 1 above)
Comment (ZaHaDumGoth) [reply files/lang/uk.po:3953]: Did not get what you mean. I have checked there is no space after sentence.

File: files/lang/uk.po (hunk 2 above)
Comment (ZaHaDumGoth) [reply files/lang/uk.po:4073]: Fixed.

File: files/lang/uk.po (hunk 1 above)
Comment (ihhub) [reply files/lang/uk.po:3953]: I meant to remove `#, fuzzy`.

Comment (ZaHaDumGoth) [reply files/lang/uk.po:3953]: Yes, sentence is from map editor, logical and correct translation.

Comment (ihhub) [reply files/lang/uk.po:3953]: Then why we are keeping **#, fuzzy** since this comment makes to skip the translation?

Comment (ZaHaDumGoth) [reply files/lang/uk.po]: Sorry, had some troubleshooting, now I got it and corrected. Clicked onto Commit and Push.

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@ZaHaDumGoth , many thanks for this update!

**Pull request merged**